runtime/near-evm-runner/src/types.rs

```diff
@@ -0,0 +1,135 @@
+use std::convert::TryInto;
+use std::io::{Read, Write};
+
+use borsh::{BorshDeserialize, BorshSerialize};
+
+use near_vm_errors::{EvmError, InconsistentStateError, VMLogicError};
+use near_vm_logic::types::AccountId;
+
+pub type RawAddress = [u8; 20];
+pub type RawHash = [u8; 32];
+pub type RawU256 = [u8; 32];
+pub type DataKey = [u8; 52];
+
+pub type Result<T> = std::result::Result<T, VMLogicError>;
+
+#[derive(BorshSerialize, BorshDeserialize)]
+pub struct AddressArg {
+    pub address: RawAddress,
+}
+
+#[derive(BorshSerialize, BorshDeserialize)]
+pub struct GetStorageAtArgs {
+    pub address: RawAddress,
+    pub key: RawHash,
+}
+
+#[derive(BorshSerialize, BorshDeserialize)]
+pub struct WithdrawArgs {
+    pub account_id: AccountId,
+    pub amount: RawU256,
+}
+
+#[derive(BorshSerialize, BorshDeserialize)]
+pub struct TransferArgs {
+    pub address: RawAddress,
+    pub amount: RawU256,
+}
+
+#[derive(Debug, Eq, PartialEq)]
```

Comment:
Don't we need `BorshSerialize, BorshDeserialize` here as well?

Reply:
https://github.com/nearprotocol/nearcore/pull/3257/files/fccae1bf033f7545331e2d38ac074574263acf9d#diff-c4090cf1331870e04bdce300c00e2db4R57